package.json

```diff
@@ -10,6 +10,9 @@
     "canvas": "yarn extender:build && yarn --cwd project-canvas dev",
     "canvas:build": "yarn extender:build && yarn --cwd project-canvas build ",
     "canvas:test": "yarn --cwd project-canvas test",
+    "canvas:eslint": "yarn --cwd project-canvas run lint",
+    "canvas:prettier": "yarn --cwd project-canvas run format",
+    "canvas:typecheck": "yarn --cwd project-canvas run typecheck",
```

Comment:
```suggestion
    "eslint": "eslint",
    "prettier": "prettier .",
    "typecheck": "tsc --noEmit",
```

Reply:
- don't include the name of the library in the command to keep it generic and not library specific
- linting and formatting should check the complete monorepo
- typechecking is tricky to run for everything at once. I suggest you run one for canvas and one for extender

Reply:
- what sould be a meaningfull name for running eslint and prettier tests so that the name of the library is not included in the command ? i suggest "eslint": "eslint",  "prettier": "prettier .",

Reply:
it should be format and lint.